asyncapi-cli/src/main/resources/listener.bal

```diff
@@ -7,11 +7,11 @@ public class Listener {
     private http:Listener httpListener;
     private DispatcherService dispatcherService;
 
-    public function init(int|http:Listener listenOn = 8090) returns error? {
-        if listenOn is http:Listener {
-            self.httpListener = listenOn;
+    public function init(int|http:Listener listenTo = 8090, *ListenerConfiguration configuration) returns error? {
```

Comment:
How are we mapping other configurations such as API key? Are they also part of this `configuration` or do we add them as required parameters in the init function?